sdks/builds/ios.mk

```diff
@@ -305,6 +305,9 @@ $(eval $(call iOSSimulatorTemplate,sim64,x86_64))
 $(eval $(call iOSSimulatorTemplate,simtv,x86_64))
 $(eval $(call iOSSimulatorTemplate,simwatch,i386))
 
+.PHONY: build-ios-llvm
+build-ios-llvm: $(if $(IGNORE_PACKAGE_LLVM),download-llvm-$(3),package-llvm-$(3))
```

Comment:
If this is only used outside mono repo, add comments? I realize that is kinda lame in general, the idea of putting comments on every line as to who uses it.

Reply:
I think that really makes sense in the sdks and makes zero sense in most other places. 
Comment added.